features/manage-position/Create.tsx

```diff
@@ -144,7 +144,7 @@ const Create = () => {
         setSuccess(null);
         setError(null);
         try {
-          const collateralWei = toWei(collateral);
+          const collateralWei = toWei(collateral, collDec);
```

Comment:
This is the other mistake in this PR: converting collateral token units to Wei requires us to explicitly set the collDec